src/org/usfirst/frc4904/robot/Robot.java

```diff
@@ -21,6 +21,7 @@ public void initialize() {
 		operatorChooser.addDefault(new DefaultOperator());
 		// Initialize SmartDashboard display values
 		// SmartDashboard.putNumber(SmartDashboardKey.EXAMPLE.key, 0);
+		RobotMap.Component.navx.zeroYaw();
```

Comment:
Oh, I remember why this bothered me: if we turn the robot on off the field, such as when the matches are behind schedule, then it will produce unexpected behaviour. I think we should add one of these to autonomousInitialize (no need to remove this one as it will still be useful during build season when we don't always use auton before teleop).